pkg/sink/sink_test.go

```diff
@@ -541,28 +536,20 @@ func TestResource_processEvent(t *testing.T) {
 	}))
 	defer ts.Close()
 
-	req, err := http.NewRequest(http.MethodPost, "http://some-url/", nil)
-	if err != nil {
-		t.Fatalf("Error trying to create request: %q", err)
-	}
+	incoming := httptest.NewRequest("POST", "http://event.listener.url", nil)
```

Comment:
"httptest.NewRequest returns a new incoming server Request, suitable for passing to an http.Handler for testing."